packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md

```diff
@@ -159,7 +159,7 @@ https://app.scaleft.com/v1/teams/${team_name}/servers
 ### Update a Server
 
 <ApiOperation method="PUT" url="https://app.scaleft.com/v1/teams/${team_name}/servers/${server_id}" />
-Updates a Server. This endpoint is only available with the PolicySync feature, which is currently in EA.
+Updates a Server. This endpoint is intended for use with the PolicySync feature.
```

Comment:
```suggestion
Updates a Server. This endpoint is used for the PolicySync feature.
```
Do you want to use the `<ApiLifecycle access="ea" />` if this feature is EA?

Reply:
It actually left EA already.